include/RED4ext/CommandLine-inl.hpp

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#ifdef RED4EXT_STATIC_LIB
+#include <RED4ext/CommandLine.hpp>
+#endif
+
+#include <RED4ext/Detail/AddressHashes.hpp>
+#include <RED4ext/Relocation.hpp>
+#include <RED4ext/StringView.hpp>
+
+RED4EXT_INLINE RED4ext::CommandLine* RED4ext::CommandLine::Get() noexcept
+{
+    // NOTE: GetAddr is actually necessary here
+
+    static CommandLine* ptr{
+        RED4ext::UniversalRelocPtr<RED4ext::CommandLine>(RED4ext::Detail::AddressHashes::CommandLine).GetAddr()};
+    return ptr;
+}
+
+RED4EXT_INLINE bool RED4ext::CommandLine::HasLaunchParameter(RED4ext::StringView aParam) noexcept
```

Comment:
i miss GetValue, some cmdline params may have options. I think that may be your unknown (pointer to output var, but would havr to check disasm...)

what is the option index for also? seems like another output or maybe in-out?

Reply:
- I don't think the unk is output param, can't get any sensible output from it in testing
- OptionIndex might not actually be option index, will rename